core/workspace_svg.ts

```diff
@@ -895,7 +895,6 @@ export class WorkspaceSvg extends Workspace implements IASTNodeLocationSvg {
     }
 
     if (this.grid_) {
-      this.grid_.dispose();
       // AnyDuringMigration because:  Type 'null' is not assignable to type
       // 'Grid'.
       this.grid_ = null as AnyDuringMigration;
```

Comment:
You don't need to set things to null anymore, but you should generally call their dispose functions so that they can unregister event listeners if they have them.

Reply:
All the dispose method in grid did was null out a field, so I just removed the method entirely since it would have been a no-op. Do we want to add it back in case somebody wants to subclass it and needs to do some cleanup?

Reply:
Ah ok I didn't catch that the method was removed entirely. I don't think it's possible to subclass this in a useful way atm (because the type being constructed is [hard coded](https://github.com/google/blockly/blob/da8e486995df8f0b7b08a086ffb1b4bd22b21963/core/workspace_svg.js#L463)). So I think the code you've got right now is good. If for some reason we want to allow people to construct subclasses in the future we can always add it back.

And luckily `dispose` is package/internal, so this isn't a breaking change.